src/runtime_src/tools/xclbinutil/SectionVenderMetadata.cxx

```diff
@@ -262,8 +254,7 @@ SectionVenderMetadata::writeObjImage(std::ostream& _oStream) const
 void
 SectionVenderMetadata::writeMetadata(std::ostream& _oStream) const
 {
-  XUtil::TRACE("VENDER_METADATA -METADATA");
-
+  XUtil::TRACE("VENDER_METADATA writeMetadata");
```

Comment:
I am curious about all these "vender". What is the meaning? Is this "vendor"?

Reply:
Hmm..you got me scared there for a moment regarding the spelling of "vender".  Did a quick look up and found the following:
```
Which is correct vendor or vender?
Vendor is British English. Vender is American English. No difference between the two other than that.

```
That said, the purpose for this section is to allow service provide to add their own content to the xclbin.  This was asked for by many provides.